dds_web/templates/user_agreement.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block body %}
+<html>
+        
+        <embed
+        src="/static/user_agreement.pdf"
+        type="application/pdf"
+        scrolling="auto"
+        height="2100px"
+        width="800px"
+        >
+        </embed>
```

Comment:
I strongly dislike how the page with embedded pdf looks. Maybe there's a way to let flask show the pdf file directly in the browser, or offer it for download. 
Alternatively, we can have the user agreement text in a html format, like the Privacy policy. 
This can be discussed with Aishling

Reply:
Yes, it can be shown directly in the browser. The localhost/static/user_agreement.pdf path in the embdeed command